pygmt/__init__.py

```diff
@@ -63,7 +62,7 @@
 )
 
 # Get semantic version through setuptools-scm
-__version__ = f'v{get_distribution("pygmt").version}'  # e.g. v0.1.2.dev3+g0ab3cd78
+__version__ = version("pygmt")  # e.g. v0.1.2.dev3+g0ab3cd78
```

Comment:
The `version` function returns the version string without the leading `v` (e.g., `0.1.2`):
```suggestion
__version__ = f'v{version("pygmt")}'  # e.g. v0.1.2.dev3+g0ab3cd78
```